lib/linguist/languages.yml

```diff
@@ -6759,6 +6759,13 @@ Wavefront Object:
   tm_scope: source.wavefront.obj
   ace_mode: text
   language_id: 393
+Whiley:
+  type: programming
+  extensions:
+  - ".whiley"
+  tm_scope: source.whiley
+  ace_mode: text
+  language_id: 888779559
```

Comment:
I recommend adding a colour else we'll fallback to a really drab grey.

Reply:
Hey,

Thanks!  I have made these changes now, including adding a colour.  I had not noticed the license file was generated and just forgot to add it to my fork.